src/ArduinoJson/Variant/VariantContent.hpp

```diff
@@ -31,7 +31,8 @@ enum {
   VALUE_IS_SIGNED_INTEGER = 0x0A,
   VALUE_IS_FLOAT = 0x0C,
 
-  COLLECTION_MASK = 0x60,
+  COLLECTION_MASK = 0x70,
```

Comment:
Collections in this context are arrays and objects, but not binary values.
Please restore the original value.

Reply:
Should I move `VALUE_IS_BINARY` out of the collections section? If so, where?

Reply:
You should use the value 0x02 which used to be reserved for linked raw strings.

Reply:
That doesn't seem right, that would make it so that binaries aren't owned by the document, which isn't true. I have it on 0x07 locally to make sure the owned flag is set right now. Is 0x02 really fine?

Reply:
You're right. It doesn't work. Sorry.

I think I oversaw this problem because my plan was to remove `VALUE_IS_BINARY` and handle binary strings with `VALUE_IS_RAW_STRING` and let the converters do all the work.
This way, we add support for MessagePack binary strings without affecting JSON users and also pave the way to support MessagePack ext values in the future.

Nevertheless, I think it's still a good stepping stone to have the `VALUE_IS_BINARY`, even if it'll soon be removed.
Please ignore my previous comment about the value, and position the enum value where it makes sense to you. Don't overthink it, since we may remove it in a future commit.